docs/diagrams/StorageClassUML.puml

```diff
@@ -0,0 +1,37 @@
+@startuml
+
+abstract List
+class PatientList
+class DoctorList
+class MedicineList
+class AppointmentList
+
+List <|-- PatientList
+List <|-- DoctorList
+List <|-- MedicineList
+List <|-- AppointmentList
```

Comment:
The arrow is too messy. Moreover, I didn't see the value to express the association between Storage and all specific XYZList, which doesn't add value. Can just show the a specific case as example